core-services/image-mirroring/hypershift/mapping_hypershift_4_10

```diff
@@ -0,0 +1 @@
+registry.ci.openshift.org/ocp/4.10:hypershift registry.ci.openshift.org/hypershift/hypershift:4.10 registry.ci.openshift.org/hypershift/hypershift:latest
```

Comment:
This also needs a corresponding job, ref https://docs.ci.openshift.org/docs/how-tos/mirroring-to-quay/#configuring-mirroring-for-new-repository

Reply:
Done